homepage/static/testimonials.json

```diff
@@ -7,11 +7,11 @@
     "text": "My laboratory produces large amounts of data from RNA-seq, ChIP-seq and genome resequencing experiments.  Orange allows me to analyze my data even though I don’t know how to program.  It also allows me to communicate with my collaborators, who are experts in data mining, and with my colleagues and trainees."
   },
   {
-    "img": "/static/images/jasna_plantan_small.jpg",
-    "title": "Jasna Plantan",
-    "position": "Data scientist and CEO",
-    "institution": "Revelo, Ljubljana, Slovenia",
-    "text": "Widget development is fun. Orange comes with all the tools for state persistency, reporting and reproducibilty. And then it is so rewarding to see your own widget working in concert with all the others."
+    "img": "/static/images/Borondics_Ferenc_small.png",
```

Comment:
The image reference changed and the new image was added, but the previous one was not removed.
Intentional? I don't know if it is used anywhere else or it might be used again in the future - in that case it is OK to keep the old one too.